backends/aiger/xaiger.cc

```diff
@@ -272,12 +250,6 @@ struct XAigerWriter
 								arrival_times[bit] = arrival;
```

Comment:
Cache arrival times so that it is not computed for every box (once for every cell type instead)